scripts/globals/weaponskills/exenterator.lua

```diff
@@ -30,8 +30,15 @@ weaponskill_object.onUseWeaponSkill = function(player, target, wsID, tp, primary
     params.acc100 = 0.0 params.acc200= 0.0 params.acc300= 0.0
     params.atk100 = 1.0; params.atk200 = 1.0; params.atk300 = 1.0
 
-    if (xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES == true) then
+    if xi.settings.USE_ADOULIN_WEAPON_SKILL_CHANGES then
         params.agi_wsc = 0.7 + (player:getMerit(xi.merit.EXENTERATOR) * 0.03)
+        params.ftp100 = 2.0 params.ftp200 = 2.0 params.ftp300 = 2.0 -- http://wiki.ffo.jp/html/25595.html
+    end
+
+    if xi.settings.USE_MULTI_HIT_FTP_WEAPON_SKILL_CHANGES then
```

Comment:
This setting is undefined, and also appears to only be duplicating the `USE_ADOULIN_WEAPON_SKILL_CHANGES` values for everything in this PR.

Reply:
it's a setting that will be added if my other ws pr goes trough.

Reply:
should I duplicate the setting here it's just one line of code in the main.lua?

Reply:
with the fTP relpication many ws ftp value became weaker like dragon kick 
from
2.00 | 3.875 | 7.00 | ×2 | STR50% VIT50%
to this
1.70 | 3.00 | 5.00 | ×2 | STR50% VIT50%
or 
exentrerator from
2.0
to 
1.1875

Reply:
I'm not too sure about this new setting, if I'm being honest.

Firstly we would have to determine what the setting is for. The adoulin setting being ON is meant for modern retail WS parameters. It being off is meant for older eras.

So if anything, I think the new setting should be removed, becouse either this parameters are the new ones, or they arent.

Reply:
Removing the setting is fine with me, I just saw the adoulin setting existed because of the big change to ws and ftp replication is another big change so I made it a choice. I just need to know what is best for this project and so I can change it accordingly.

Reply:
adoulin one is basically mis-named. it should say "modern" instead of "adoulin" and the aim is for an easy toggle for the old school crowd who wants 75 cap gameplay and the rest of us, who are matching modern retail (by default).

Reply:
New setting was removed.